packages/dtslint/src/index.ts

```diff
@@ -273,10 +275,15 @@ async function testTypesVersion(
   const errors = [];
   const checkExpectedFilesResult = checkExpectedFiles(dirPath, isLatest);
   errors.push(...checkExpectedFilesResult.errors);
-  const tsconfigErrors = checkTsconfig(dirPath, getCompilerOptions(dirPath));
-  if (tsconfigErrors.length > 0) {
-    errors.push("\n\t* " + tsconfigErrors.join("\n\t* "));
+
+  for (const tsconfig of tsconfigs) {
+    const tsconfigPath = joinPaths(dirPath, tsconfig);
+    const tsconfigErrors = checkTsconfig(dirPath, getCompilerOptions(tsconfigPath));
+    if (tsconfigErrors.length > 0) {
+      errors.push("\n\t* " + tsconfigErrors.join("\n\t* "));
```

Comment:
Is it clear from the error text which tsconfig this applies to?

Reply:
Nope, definitely not, will fix.

Reply:
Modified this to include a filename before each of these groups.